hs/src/Reach/AST/DK.hs

```diff
@@ -104,10 +104,10 @@ type DKExports = M.Map SLVar DKExportBlock
 type DKViews = DLViews
 
 data DKProg
-  = DKProg SrcLoc DLOpts SLParts DLInit DKExports DKViews DLAPIs DLEvents DKTail
+  = DKProg SrcLoc DLOpts SLParts DLInit DKExports DKViews DLAPIs Aliases DLEvents DKTail
```

Comment:
Can you make a Jira card to turn all of the `*Prog` datas to records so we can name these things?